### PR TITLE
Leadspace

### PR DESCRIPTION
**Pull request by andysherman2121**
### Related Ticket(s)

https://github.com/carbon-design-system/ibm-dotcom-library/issues/1115
https://github.com/carbon-design-system/ibm-dotcom-library/issues/1184
https://github.com/carbon-design-system/ibm-dotcom-library/issues/1186
https://github.com/carbon-design-system/ibm-dotcom-library/issues/1182

### Description

Consolidates LeadSpace and LeadSpace centered into a single component

### CODE REVIEW

**Comment by jeffchew** [comment]
@andysherman2121 Has `LeadSpaceCentered` been deleted as part of this PR?

@annawen1 can you take a look at this PR when you get a chance?

File: packages/react/src/patterns/sections/LeadSpace/LeadSpace.js [hunk 1]
```diff
@@ -25,17 +21,62 @@ const { prefix } = settings;
...
+    return (
+      <div
+        data-autoid={`${stablePrefix}--leadspace--centered--mobile__image`}
+        className={`${prefix}--leadspace--centered--mobile__image`}>
+        <img src={image.default} alt={image.alt} />
+      </div>
+    );
+  } else
+    return (
+      <LeadSpaceImage
```
Comment (annawen1) [inline packages/react/src/patterns/sections/LeadSpace/LeadSpace.js]: We can actually use the `Image` component here now instead and remove `LeadSpaceImage`

File: packages/react/src/patterns/sections/LeadSpace/LeadSpace.js [hunk 2]
```diff
@@ -90,43 +131,72 @@ const LeadSpace = ({
...
   theme,
   title,
+  type,
   variation,
-}) =>
-  featureFlag(
-    DDS_LEADSPACE,
+}) => {
+  const [windowWidth, setWindowWidth] = useState(window.innerWidth);
+  const isMobile = windowWidth <= 671;
```
Comment (annawen1) [inline packages/react/src/patterns/sections/LeadSpace/LeadSpace.js]: we can also use carbon's breakpoint values here too like in the `Image` component
```
import { baseFontSize, breakpoints } from '@carbon/layout';

....

  const isMobile = windowWidth <= parseFloat(breakpoints.sm.width * baseFontSize);
```

...

**Comment by annawen1** [comment]
@andysherman2121 didn't you resolve this already?

Screen Shot 2020-02-25 at 5 08 26 PM

**Comment by andysherman2121** [comment]
> @andysherman2121 didn't you resolve this already?
> 
> Screen Shot 2020-02-25 at 5 08 26 PM

yupp, it's fixed in the other PR that's upcoming

**Comment by jeffchew** [comment]
@annawen1 @andysherman2121 the README needs updating for the Leadspace (update `variations`, remove the feature flag, etc).

I also noticed the latest deploy preview isn't rendering the ButtonGroup.

**Pull request merged**